lib/rules/require-jsdoc.js

```diff
@@ -23,22 +23,34 @@ module.exports = {
                         type: "object",
                         properties: {
                             ClassDeclaration: {
-                                type: "boolean"
+                                type: "boolean",
+                                default: false
                             },
                             MethodDefinition: {
-                                type: "boolean"
+                                type: "boolean",
+                                default: false
                             },
                             FunctionDeclaration: {
-                                type: "boolean"
+                                type: "boolean",
+                                default: true
                             },
                             ArrowFunctionExpression: {
-                                type: "boolean"
+                                type: "boolean",
+                                default: false
                             },
                             FunctionExpression: {
-                                type: "boolean"
+                                type: "boolean",
+                                default: false
                             }
                         },
-                        additionalProperties: false
+                        additionalProperties: false,
+                        default: {
```

Comment:
This is interesting-- we have an object default and property defaults. Is it possible for this default to be `{}` and then Ajv would recursively apply the property schema defaults? Or will that not work?

Reply:
I'm not sure, but I'll do a test and reply to you.